src/c64/mmu.cpp

```diff
@@ -27,6 +27,17 @@
 
 namespace libsidplayfp
 {
+// Use static type to allow compiler to optimize virtual function call
+template<class BankType, BankType MMU::* Bank>
+uint8_t Read(MMU &self, uint_least16_t addr)
```

Comment:
Can you please rename the functions to start with a lowercase, just for consistency?

Reply:
Done